index.html

```diff
@@ -304,6 +305,291 @@ <h2>Contact me</h2>
+                </button>
+                <button type="button" class="modal-project">See Source
+                    <img src="asset/Vector.svg" alt="git_logo">
+                </button>
+                </div>
+            </div>
+        </div>
+    </div>
+
+        <!-- 2ND WORK SECTION -->
+    <div class="modal" id="modal2">
+        <div class="modal-box">
+            <span class="closeBtn" data-modal="modal2">&times;</span>
+            <div class="modal-info">
+                <h3 class="modal-title">Multi-Post Stories</h3>
+                <div class="modal-meta">
+                    <p class="modal-para">facebook</p>
+                    <img src="asset/Counter.svg" alt="dot">
+                    <p class="modal-para1">Full Stack Dev</p>
+                    <img src="asset/Counter.svg" alt="dot">
+                    <p class="modal-para2">2015</p>
+                </div>
+            </div>
+            <div class="modal-img">
+                <img src="asset/Snapshoot-Portfolio.svg" alt="work card project">
+            </div>
+                <div class="modal-note">
+                    <p>Lorem Ipsum is simply dummy text of the printing 
+                        and typesetting industry. Lorem Ipsum has been 
+                        the industry's standard dummy text ever since the 
+                        1500s, when an unknown printer took a galley of type 
+                        and scrambled it to make a type specimen book.</p>
+                    <p>Lorem Ipsum is simply dummy text of the printing 
+                        and typesetting industry. Lorem Ipsum has been 
+                        the industry's standard dummy text ever since the 
+                        1500s, when an unknown printer took a galley of 
+                        type and scrambled it 1960s with the releaLorem 
+                        Ipsum is simply dummy text of the printing and 
+                        ypesetting industry. Lorem Ipsum has been the 
+                        industry's standard dummy text ever since the 
+                        1500s, when an unknown printer took a galley of 
+                        type and scrambled it 1960s with the releorem 
+                        Ipsum is simply dummy text of the printing and 
+                        typesetting industry. Lorem Ipsum han printer 
+                        took a galley of type and scrambled it 1960s with 
+                        the releawn printer took a galley of type and 
+                        scrambled it 1960s with the releaLorem Ipsum is 
+                        simply dummy text of the printing and typesetting 
+                        industry. Lorem Ipsum has been the industry's 
+                        standard dummy text ever since the 1500s, when 
+                        an unknown printer took a galley of type and 
+                        scrambled it 1960s with the relea</p>
+                </div>
```

Comment:
- [ ] The popup's picture in desktop mode requires a couple of tweaks before calling them ready. Kindly make sure to follow the Figma design.
## FIGMA
![figma popup](https://user-images.githubusercontent.com/69858269/169399915-ea58b87a-8d47-489b-9d14-420579caeaf9.PNG)
## Yours
![your popup](https://user-images.githubusercontent.com/69858269/169399998-1f687a83-8351-4e34-b929-42ccabae3c9a.PNG)

Reply:
resolved
thanks